--- "Cgl\\configure"	2016-02-22 20:20:13.000000000 +0100
+++ "Cgl\\configure.new"	2017-03-17 21:19:50.725571300 +0100
@@ -20838,6 +20838,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -21354,6 +21355,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -21859,6 +21861,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -22352,6 +22355,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -22860,6 +22864,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -23340,6 +23345,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -23821,6 +23827,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -24301,6 +24308,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -24781,6 +24789,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -25261,6 +25270,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
