--- "CoinUtils\\configure"	2016-02-11 06:41:12.000000000 +0100
+++ "CoinUtils\\configure.new"	2017-03-17 21:22:33.532101900 +0100
@@ -32608,6 +32608,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -35207,6 +35208,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -35768,6 +35770,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -36267,6 +36270,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -36747,6 +36751,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
