--- "Osi\\configure"	2016-02-22 19:26:30.000000000 +0100
+++ "Osi\\configure.new"	2017-03-17 21:22:54.279609100 +0100
@@ -20836,6 +20836,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -21351,6 +21352,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -21861,6 +21863,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -22360,6 +22363,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
@@ -22840,6 +22844,7 @@
     # need to export variable to be sure that the following pkg-config gets these values
     coin_save_PKG_CONFIG_PATH="$PKG_CONFIG_PATH"
     PKG_CONFIG_PATH="$COIN_PKG_CONFIG_PATH:$COIN_PKG_CONFIG_PATH_UNINSTALLED"
+    PKG_CONFIG_PATH=`echo $PKG_CONFIG_PATH | sed -e 's/:/;/g' | sed -e 's/C;/C:/g' | sed -e 's/\/c\//C:\//g'`
     export PKG_CONFIG_PATH
 
     # let pkg-config do it's magic
